samza-test/src/main/resources/log4j.xml

```diff
@@ -26,14 +26,14 @@
     <param name="MaxFileSize" value="256MB" />
     <param name="MaxBackupIndex" value="1" />
     <layout class="org.apache.log4j.PatternLayout">
-      <param name="ConversionPattern" value="%d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} [%p] %m%n" />
```

Comment:
Not clear why the logging format is changing based on the pull request description.